include/rest/api_client.hpp

```diff
@@ -144,19 +243,11 @@ namespace disccord
                     }
 
                     template <typename TModel>
-                    pplx::task<void> request_multi_json(route_info& route, std::vector<TModel> body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
+                    pplx::task<void> request_multi_json(route_info& route, TModel body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
```

Comment:
yeah thats the thing tho, the actual `rest::model` isnt going to be a vector, but when you eventually call its `encode_to` override, the result will be a json array. 

If you look at the example of one of the requests I made in this specific commit it will make more sense. But essentially, the only requests where the user will have to build the actual `rest::model` themselves are ones where there are optional parameters. but for the 2 endpoints that require an array, the are mandatory. so in these cases I have the user only build a vector, and pass that in to a model that has that vector as a member, instead of making the user build a model for each one, since we know exactly what they have to pass in, we can do that work for them instead.